index.bs

```diff
@@ -1150,8 +1150,8 @@ interface ReadableStreamDefaultReader {
 ReadableStreamDefaultReader includes ReadableStreamGenericReader;
 
 dictionary ReadableStreamDefaultReadResult {
- any value;
- boolean done;
+  any value;
```

Comment:
Might be worth pulling the Web IDL and editorial changes out into a separate PR that we can just approve and merge ASAP.

Reply:
Fair point, opened #1192.

Reply:
Dangit, I probably should have moved https://github.com/whatwg/streams/pull/1190/commits/4fd1b3badcd36bb45b78b7f226a8828002973da0 out to that PR too. 🤦‍♂️